cmd/runsim/notification.go

```diff
@@ -2,82 +2,101 @@ package main
 
 import (
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
this makes sense if you intend to use the `Wrap()` function.
Alternatively I'd suggest to depend on the standard `errors` package.

Reply:
oops, that was just Goland auto importing stuff. Fixed.